website/versioned_docs/version-0.19.0/tutorial/index.mdx

```diff
@@ -367,7 +367,7 @@ Then we modify the `VideosList` component to pass the "emit" the selected video
 ```rust ,ignore {5-11,14-15}
```

Comment:
The highlighting is applied here, too. Can you add line 3 so it gets a highlight too?

Reply:
Absolutely, I will get on it now. Thanks for pointing out the location to update.

Reply:
Can you address the other part, i.e. also applying the changes to `docs/tutorial/index.mdx` - which will become `version-0.20.0/tutorial/index.mdx` when we release? Otherwise the changes will be lost to time when we at some point remove the tutorial for 0.19.

Reply:
Hey, I made the necessary changes. Thanks